external/anomaly/ote_tests_pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+python_files = test_ote_task.py
```

Comment:
@LeonidBeynenson  are you ok with this?

Reply:
Sorry, no.
@AlexanderDokuchaev : why do you set it here?

Reply:
It will be removed. 
I have tried to run tests in ci